README.md

```diff
@@ -121,19 +120,26 @@ After do |scenario|
     # Can be used to do specific cleanup if a scenario fails
   end
 end
- ```
+```
+
+### Step definition links
 
-### Step reference
+- [Request assertion steps](/maze-runner/requirements/step_transformers.html#step_definition64-stepdefinition)
```

Comment:
Are these links meant to work from the github.com/bugsnag/maze-runner page? I'm not sure they will 🤔

Also, are they likely to change if new steps are added?

Reply:
For the first point, no, and I'll remedy that by separating the two (i.e. Readme for github and front-page for docs).

For the second point, yes, and without amending the plugin template to add groups there doesn't seem to be much we can do other than leave them out entirely.  It's not great.

Reply:
Sure - it'll do for now.